network-api/networkapi/settings.py

```diff
@@ -66,6 +66,7 @@
     SLACK_WEBHOOK_RA=(str, ''),
     BUYERS_GUIDE_VOTE_RATE_LIMIT=(str, '200/hour'),
     CORAL_TALK_SERVER_URL=(str, ''),
+    PNI_STATS_DB_URL=(str, ''),
```

Comment:
Probably better to default to `None` so that the `setup_db_connection(self)` function above can test for `if STATS_DB is None: ...`?